build/distributions/CentOS/7/x86_64/DVD/7-simp_pkglist.txt

```diff
@@ -1850,17 +1493,121 @@ yum
 yum-metadata-parser
 yum-plugin-aliases
 yum-plugin-changelog
-yum-plugin-downloadonly
 yum-plugin-fastestmirror
-yum-plugin-security
 yum-plugin-tmprepo
 yum-plugin-verify
 yum-plugin-versionlock
-yum-presto
-yum-rhn-plugin
 yum-utils
 zd1211-firmware
 zenity
 zip
 zlib
 zlib-devel
+authconfig
```

Comment:
why was this moved down?  It is much easier to understand this package list if it is alphabetical